app/src/main/java/com/work/restaurant/util/DateAndTime.kt

```diff
@@ -0,0 +1,165 @@
+package com.work.restaurant.util
+
+import android.annotation.SuppressLint
+import java.text.SimpleDateFormat
+import java.util.*
+
+@SuppressLint("SimpleDateFormat")
+object DateAndTime {
+
+    private const val datePattern = "yyyy년 M월 d일"
+    private val simpleDateFormat = SimpleDateFormat(datePattern)
+
+    fun currentDate(): String =
+        simpleDateFormat.format(Date())
+
+    fun convertDate(date: String): List<String> {
+
+        val list = mutableListOf<String>()
+
+        date.split(" ").apply {
+            this.forEach {
+                list.add(Regex("[^0-9]").replace(it, ""))
+            }
+        }
```

Comment:
```suggestion
        return date.split(" ").map {
                Regex("[^0-9]").replace(it, "")
        }
```
요런 식으로 개선해볼 수 있지 않을까 합니다.

Reply:
37305a0 반영하였습니다.